vaadin-date-time-picker-flow-testbench/src/main/java/com/vaadin/flow/component/datetimepicker/testbench/DateTimePickerElement.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2000-2019 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.datetimepicker.testbench;
+
+import java.time.LocalDateTime;
+
+import com.vaadin.testbench.HasLabel;
+import com.vaadin.testbench.TestBenchElement;
+import com.vaadin.testbench.elementsbase.Element;
+
+/**
+ * A TestBench element representing a <code>&lt;vaadin-date-time-picker&gt;</code>
+ * element.
+ */
+@Element("vaadin-date-time-picker")
+public class DateTimePickerElement extends TestBenchElement implements HasLabel {
+
+    /**
+     * Clears the value of the date time picker.
+     */
+    @Override
+    public void clear() {
+        setDateTime(null);
+    }
+
+    /**
+     * Selects the given date.
```

Comment:
Fix all the JavaDocs in this file from `date` to `date time`.

Reply:
Done.